packages/rocketchat-livechat/server/models/Rooms.js

```diff
@@ -176,6 +176,30 @@ RocketChat.models.Rooms.saveAnalyticsDataByRoomId = function(roomId, inc, analyt
 	}, update);
 };
 
+RocketChat.models.Rooms.setLastMessageTimeById = function(room, message) {
+	const update = {
+		$set: {}
+	};
+
+	// livechat analytics : update last message timestamps
+	if (room.t === 'l') {
```

Comment:
It seems strange checking `room.t === 'l'` within Livechat Room model.
You have to check this outside the model, before calling the method.